src/codegen/formatter/tests.rs

```diff
@@ -226,15 +227,72 @@ class CarFoo {
 
 #[test]
 fn simple_module() {
-    let formatted_code = "module main
+    let formatted_code = format_elz("module main
 
-main(): void {}
-";
+main(): void {
+  println(\"Hello, World!\");
+}
+".to_string()
+    );
     assert_eq!(
         formatted_code,
         "module main
 
-main(): void {}
+main(): void {
+  println(\"Hello, World!\");
+}
 "
     );
 }
+
+#[test]
+fn multi_class() {
+    let formatted_code = format_elz("module prelude
```

Comment:
We don't need this test! Testing is not copied failure cases, but those help people understand why code is correct.